.github/workflows/formatting.yaml

```diff
@@ -3,15 +3,24 @@ name: OpenTurbine-CI
 on: push
 
 jobs:
-  ClangFormat:
+  clang-format:
     runs-on: ubuntu-latest
+
```

Comment:
These empty lines are not used in rest of the CI workflows, just pointing it out.